csharp-tree-sitter.el

```diff
@@ -8,7 +8,7 @@
 ;; Version    : 0.11.0
 ;; Keywords   : c# languages oop mode
 ;; X-URL      : https://github.com/emacs-csharp/csharp-mode
-;; Package-Requires: ((emacs "26.1") (tree-sitter "0.12.1") (tree-sitter-indent "0.1") (tree-sitter-langs "0.9.0"))
+;; Package-Requires: ((emacs "26.1") (csharp-mode "0.11.0") (tree-sitter "0.12.1") (tree-sitter-indent "0.1") (tree-sitter-langs "0.9.0"))
```

Comment:
I am turning this package to `minor-mode` hence it should just depends on `csharp-mode` itself.